worker/agentconfigupdater/manifold.go

```diff
@@ -39,7 +48,8 @@ func Manifold(config ManifoldConfig) dependency.Manifold {
 			}
 
 			// Grab the tag and ensure that it's for a machine.
-			tag, ok := agent.CurrentConfig().Tag().(names.MachineTag)
+			currentConfig := agent.CurrentConfig()
```

Comment:
What's wrong with inlining? That's the idiomatic Go way to do it.

Reply:
Because I use currentConfig further down the method.